packages/react/src/components/Table/StatefulTable.jsx

```diff
@@ -242,6 +244,8 @@ const StatefulTable = ({ data: initialData, expandedData, ...other }) => {
       onDownloadCSV,
     },
     table: {
+      onDrag,
+      onDrop,
```

Comment:
OK, I changed these to use `callbackParent` which protects against null callbacks. Mostly its for consistency though. There's nothing to `dispatch` since none if this state is in the reducer state, similar to `onFilterRowIconClick`.